src/test/java/com/wikia/webdriver/elements/mercury/pages/ArticlePage.java

```diff
@@ -0,0 +1,45 @@
+package com.wikia.webdriver.elements.mercury.pages;
+
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.elements.common.Navigate;
+import com.wikia.webdriver.elements.mercury.components.Loading;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ArticlePage extends WikiBasePageObject {
+
+  @FindBy(css = "article a[href=\"/wiki/Category:Category_page_test"
```

Comment:
First `"` is required by Java to create String, second `"` is not replaceable in the selector (can be omitted, but it's safer to use it).